modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RubyClientCodegen.java

```diff
@@ -141,6 +143,18 @@ public RubyClientCodegen() {
         cliOptions.add(new CliOption(CodegenConstants.HIDE_GENERATION_TIMESTAMP, CodegenConstants.HIDE_GENERATION_TIMESTAMP_DESC).
                 defaultValue(Boolean.TRUE.toString()));
 
+        cliOptions.add(CliOption.newBoolean(FARADAY, "Faraday (https://github.com/lostisland/faraday)"));
+        cliOptions.add(CliOption.newBoolean(TYPHOEUS, "Typhoeus (https://github.com/typhoeus/typhoeus)"));
```

Comment:
@dkliban I don't think we need line 146, 147 given that we have the library option.